proto/viam/app/v1/app.proto

```diff
@@ -603,10 +603,20 @@ message GetRobotPartResponse {
 }
 
 message GetRobotPartLogsRequest {
+  enum LogLevel {
```

Comment:
can you move this out of the message for LogsRequest? It may be useful for other requests

Reply:
Done